packages/components/src/Page/Page.tsx

```diff
@@ -62,25 +63,60 @@ export function Page({
   moreActionsMenu = [],
 }: PageProps) {
   const pageStyles = classnames(styles.page, styles[width]);
+  const [titleBarRef, { width: titleBarWidth }] = useResizeObserver<
+    HTMLDivElement
+  >();
+
+  const titleBarClasses = classnames(styles.titleBar, {
+    [styles.small]: titleBarWidth && titleBarWidth > 265,
+    [styles.medium]: titleBarWidth && titleBarWidth > 500,
+    [styles.large]: titleBarWidth && titleBarWidth > 750,
```

Comment:
I think we should add, to the followup, a task to formalize these breakpoints.

Reply:
I have added a task to do this